src/common/i18n.js

```diff
@@ -44,9 +44,11 @@ export const translate = (input, params = []) => {
         const stringToBeTranslated = input.replace(/\{\$({0-9])\}/gi, '%$1');
         let translatedString = i18nTranslate(stringToBeTranslated);
         params.forEach((replacement, index) => {
-            translatedString = translatedString.replaceAll(`\{\$${index}\}`, replacement);
+            if (translatedString && typeof translatedString === 'string') {
+                translatedString = translatedString.replaceAll(`\{\$${index}\}`, replacement);
+            }
         });
-        return RenderHTML(translatedString);
+        return RenderHTML(translatedString || '');
```

Comment:
Sounds like having both `translatedString &&` and `translatedString || ''` is redundant here.
Am I correct? Could we remove one and keep just the other?

Reply:
Removed.